unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/WorldChatWindowHUD/IChannelChatWindowView.cs

```diff
@@ -9,6 +9,7 @@ public interface IChannelChatWindowView
     bool IsActive { get; }
     IChatHUDComponentView ChatHUD { get; }
     RectTransform Transform { get; }
+    string channel { get; }
```

Comment:
lets follow CamelCase c# standard naming: `Channel`

Reply:
Done in https://github.com/decentraland/unity-renderer/pull/2257/commits/90718932c80c543137a3c147edae8b44cf83c2e0